user_docs/en/userGuide.t2t

```diff
@@ -2093,6 +2100,39 @@ Please check your BrailleNote's documentation to find where these keys are locat
 | Windows key | space+dot2+dot4+dot5+dot6 (space+w) |
 | Alt key | space+dot1+dot3+dot4 (space+m) |
 | Toggle input help | space+dot2+dot3+dot6 (space+lower h) |
+
+Following are commands assigned to BrailleNote QT.
+%kc:beginInclude

```

Comment:
1. A kc include block was already started earlier.
2. You probably want to include the sentence explaining the purpose of each table in the kc document anyway, so just leave the kc include block open from before.
3. Should the braille dot commands be split out of the main commands into a separate table, since I assume you can't execute those on a QT?